com.microsoft.mrtk.input/Controllers/ArticulatedHandController.cs

```diff
@@ -162,14 +162,14 @@ protected override void UpdateTrackingInput(XRControllerState controllerState)
                 }
 
                 // Tick the hand ray generator function. Uses index knuckle for position.
-                handRay.Update(PlayspaceUtilities.ReferenceTransform.TransformPoint(knuckle.Position),
-                               PlayspaceUtilities.ReferenceTransform.TransformVector(-palm.Up),
-                               CameraCache.Main.transform,
-                               HandNode.ToHandedness());
+                handRay.Update(knuckle.Position, -palm.Up, CameraCache.Main.transform, HandNode.ToHandedness());
                 
                 Ray ray = handRay.Ray;
-                controllerState.position = ray.origin;
-                controllerState.rotation = Quaternion.LookRotation(ray.direction, PlayspaceUtilities.ReferenceTransform.TransformVector(palm.Up));
+                
+                // controllerState is in rig-local space, our ray generator works in worldspace!
+                controllerState.position = PlayspaceUtilities.ReferenceTransform.InverseTransformPoint(ray.origin);
+                controllerState.rotation = Quaternion.LookRotation(PlayspaceUtilities.ReferenceTransform.InverseTransformVector(ray.direction),
```

Comment:
Would we want to use `InverseTransformDirection` for `ray.direction`?

Reply:
I beliiiiiiiiiiiiiiieeve it's equivalent in this context 🙃

Reply:
Ahhhh yeah, I think you're right, since we're just turning it into a `LookRotation` anyway! Just might be doing more work than needed, since `InverseTransformVector` takes into account scale, but that feels like it should be fine